notification-api-service/src/main/java/com/smarsh/notificationservice/api/NotificationServiceRunner.java

```diff
@@ -14,5 +14,6 @@
 
     public static void main(String[] args) {
         SpringApplication.run(NotificationServiceRunner.class, args);
+        throw new RuntimeException();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://internal-fake-sonarqube-elb-2-1800924245.us-west-2.elb.amazonaws.com/coding_rules#rule_key=squid%3AS00112)